scripts/pre_commit_linter.py

```diff
@@ -1357,6 +1368,63 @@ def _check_html_indent(all_files, debug=False):
     return summary_messages
 
 
+def _check_for_copyright_notice(all_files):
+    """This function checks whether the copyright notice
+    is present at the beginning of files.
+    """
+
+    js_files_to_check = [
+        filename for filename in all_files if filename.endswith('.js') and (
+            not filename.endswith(GENERATED_FILE_PATHS)) and (
+                not filename.endswith(CONFIG_FILE_PATHS))]
+    py_files_to_check = [
+        filename for filename in all_files if filename.endswith('.py') and (
+            not filename.endswith('__init__.py'))]
+    sh_files_to_check = [
+        filename for filename in all_files if filename.endswith('.sh')]
+    all_files_to_check = (
+        js_files_to_check + py_files_to_check + sh_files_to_check)
+    regexp_to_check = (
+        r"Copyright \d{4} The Oppia Authors\. All Rights Reserved\.")
```

Comment:
Wrap this in single quotes.

Reply:
We have enclosed all our regexp in double quotes in `BAD_PATTERNS_JS_REGEXP`, `BAD_LINE_PATTERNS_HTML_REGEXP` and `BAD_PATTERNS_PYTHON_REGEXP`. Is there some convention for it?

Reply:
Hmm, actually there is. The official Python [docs](https://docs.python.org/2/library/re.html#re.sub) and the Google Style [Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#310-strings) recommends using single quotes, unless the regex is multi-line.

Reply:
Changed quotes here and at other places for regexp.